src/kibali/PermissionsStubGenerator.cs

```diff
@@ -35,14 +53,145 @@ public string GenerateTable()
+        {
+            var resource = resourceEntry.Value;
+            if (!string.IsNullOrEmpty(this.method) && resource.SupportedMethods.TryGetValue(this.method, out var supportedSchemes))
+            {
+                var scopesByScheme = resource.FetchTableScopesByScheme(this.method, supportedSchemes);
+                MergeTableScopes(scopesByScheme, mergedTableScopes);
+            }
+        }
+        if (mergedTableScopes.Count == 0)
+        {
+            return table;
+        }
+
+        var groupedScopesByScheme = GroupAndCleanScopes(mergedTableScopes);
+        table = TableGenerator.GeneratePermissionsTable(groupedScopesByScheme);
+        return table;
+    }
+
+    void MergeTableScopes(Dictionary<string, (string, string)> scopesByName, Dictionary<string, Dictionary<string, SortedSet<string>>> mergedTableScopes)
+    {
+        foreach (var scheme in scopesByName)
+        {
+            var least = scheme.Value.Item1;
+            var higher = scheme.Value.Item2.Split(',').Select(k => k.Trim());
+            if (mergedTableScopes.TryGetValue(scheme.Key, out Dictionary<string, SortedSet<string>> mergedScopes))
+            {
+                mergedScopes["least"].Add(least);
+                mergedScopes["higher"].UnionWith(higher);
+            }
+            else
+            {
+                mergedTableScopes[scheme.Key] = new() { { "least", new() { least } }, { "higher", new SortedSet<string>(higher) } };
+            }
+        }
+    }
+
+    Dictionary<string, (string, string)> GroupAndCleanScopes(Dictionary<string, Dictionary<string, SortedSet<string>>> mergedTableScopes)
+    {
+        var notPresent = new[] { StringConstants.PermissionNotSupported, StringConstants.PermissionNotAvailable };
+        var groupedSchemePermissions = new Dictionary<string, (string, string)>();
+        foreach (var schemeEntry in mergedTableScopes)
+        {
+            var scheme = schemeEntry.Key;
+            var least = schemeEntry.Value["least"];
+            string mergedLeast;
+            if (least.Count == 1)
+            {
+                mergedLeast = least.First();
+            }
+            else
+            {
+                var filtered = least.Except(notPresent);
+                if (filtered.Any())
+                {
+                    if (filtered.Count() > 1)
+                    {
+                        mergedLeast = filtered.OrderBy(m => m).First();
+                        // TODO: Track these cases
+                        ////throw new ArgumentException("Unable to merge least privilege permissions.");
+                    }
+                    else
+                    {
+                        mergedLeast = filtered.First();
+                    }
+                }
+                else
+                {
+                    mergedLeast = StringConstants.PermissionNotSupported;
+                }
+            }
+
+            var higher = schemeEntry.Value["higher"];
```

Comment:
```suggestion
            var higher = value["higher"];
```
Depends on previous suggestion.